package.json

```diff
@@ -14,10 +14,10 @@
 		"test:linux": "node ./run-on-linux.js",
 		"build": "composer install --optimize-autoloader --prefer-dist && composer install --no-scripts --no-dev && composer dump-autoload && grunt build",
 		"setup-templates": "rm -rf ./templates && mkdir -p ./templates/playwright && mkdir -p ./templates/lighthouse && cp ./tests/playwright/templates/* ./templates/playwright && cp ./tests/lighthouse/templates/* ./templates/lighthouse",
-		"start-local-server": "npm run setup-templates && wp-env start",
+		"start-local-server": "npm run setup-templates && wp-lite-env start config=./tests/playwright/.playwright-wp-lite-env.json -- port=8888 && wp-lite-env start config=./tests/playwright/.playwright-wp-lite-env.json -- port=8889",
 		"test:php": "docker-compose run --rm wordpress_phpunit phpunit",
 		"test:php_multisite": "docker-compose run -e WP_MULTISITE=1 --rm wordpress_phpunit phpunit",
-		"test:setup:playwright": "wp-env run cli bash elementor-config/setup.sh && wp-env run tests-cli bash elementor-config/setup.sh",
+		"test:setup:playwright": "wp-lite-env cli config=./tests/playwright/.playwright-wp-lite-env.json -- port=8888 command=\"bash elementor-config/setup.sh\" && wp-lite-env cli config=./tests/playwright/.playwright-wp-lite-env.json -- port=8889 command=\"bash elementor-config/setup.sh\"",
```

Comment:
is this the setup only for CI? or is it for local dev and CI?

Reply:
Not sure what you mean, it should work both locally and in the CI